.travis.yml

```diff
@@ -29,9 +29,8 @@ before_install:
 install:
   # Customise the testing environment
   # ---------------------------------
+  - conda install --channel ioos "Django>=1.7,<1.8" python-dateutil "matplotlib>=1.2.0" "netCDF4>=1.0.2" "rtree==0.7.0" jsonfield pytz djangorestframework pyugrid pyproj "pyaxiom>=0.0.9"
```

Comment:
You can use the `requirements.txt` here saving you the time of keep two files in sync:

`conda install --file requeriments.txt`